src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,261 @@
+        long reserveId = 0L;
+        System.out.println("Do you want to make a reservation?");
+        VisualUserMenu.printListInConsole(ListMenu.getBookingHeader(), ListMenu.getBookingMenu());
+        Integer action;
+        do {
+            action = Integer.parseInt(visualUserMenu.getValidInputFromUser("Choose action: ", InputType
+                .INTEGER));
+            switch (action) {
+                case (1):
+                    Integer hotelIndex;
+                    Integer roomIndex;
+                    do {
+                        hotelIndex = Integer.parseInt(visualUserMenu.getValidInputFromUser("Enter the number of "
+                            + "hotel: ", InputType.INTEGER));
+                    } while (!visualUserMenu.validateIntegerSize(foundHotelNames.size(), hotelIndex));
+                    String selectedHotelName = foundHotelNames.get(hotelIndex - 1);
+                    do {
+                        roomIndex = Integer.parseInt(visualUserMenu.getValidInputFromUser("Enter the number of "
+                           + "room: ", InputType.INTEGER));
+                    } while (!visualUserMenu.validateIntegerSize(foundRooms.get(selectedHotelName).size(), roomIndex));
+                    Hotel selectedHotel = hotelController.findHotelByName(selectedHotelName).get(0);
+                    Room selectedRoom = foundRooms.get(selectedHotelName).get(roomIndex - 1);
+                    reserveId = reservationController.bookRoom(reservedFrom,
+                        reservedTo, selectedRoom, selectedHotel);
+
+                    System.out.println("Room reserved successfully!\n You reservation ID: " + reserveId);
+                    break;
+                default:
+                    return;
+            }
+        } while (!visualUserMenu.validateIntegerSize(ListMenu.getBookingMenu().size(), action));
+    }
+
+    private void getAllUserReservationsMenu() {
+        List<Reservation> reservations = reservationController.getAllUserReservations();
+        if (reservations.isEmpty()) {
+            System.out.println("No reservations found");
+            return;
+        }
+        VisualUserMenu.printListInConsole(null, reservations);
+        cancelReservationMenu(reservations);
+    }
+
+    private void cancelReservationMenu(List<Reservation> reservations) {
+        Integer reservationIndex;
+        do {
+            reservationIndex = Integer.parseInt(visualUserMenu.getValidInputFromUser("Enter your reservation number "
+                + "(enter "
+                + "\"0\" to return back to menu): ", InputType
+                .INTEGER));
+            if (reservationIndex == 0) {
+                return;
+            }
+        } while (!visualUserMenu.validateIntegerSize(reservations.size(), reservationIndex));
+        reservationController.cancelReservation(reservations.get(reservationIndex - 1).getId());
+        System.out.println("Reservation successfully cancelled");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)